internal/plugin.go

```diff
@@ -114,65 +114,92 @@ func (p *Plugin) Generate() ([]byte, error) {
 		}
 	}
 
-	// Keep track of which placement maps to which policy. This will be used to determine
+	// Keep track of which placement maps to which policy and policySet. This will be used to determine
 	// how many placement bindings are required since one binding per placement is required.
-	plcNameToPolicyIdxs := map[string][]int{}
+	// plcNameToPolicyAndSetIdxs[plcName]["policy"] stores the index of policy
+	// plcNameToPolicyAndSetIdxs[plcsetName]["policyset"] stores the index of policyset
```

Comment:
What if the policy and policy set have the same name?

Reply:
This is a typo. It should be plcNameToPolicyAndSetIdxs[plc~~set~~Name]["policyset"]. Both array will be accounted when placement is being generated on line 160